src/artemis/fast_grid_scan_plan.py

```diff
@@ -122,6 +128,9 @@ def gridscan_with_communicator(fgs_comp, det, params):
         fgs_composite.sample_motors, communicator.xray_centre_motor_position
     )
 
+    # tidy up
+    yield from set_zebra_shutter_to_manual(fgs_composite.zebra)
```

Comment:
Should: We probably need to do this as a `finally`. I think something like [this](https://nsls-ii.github.io/bluesky/generated/bluesky.preprocessors.finalize_decorator.html?highlight=finalizer) will help

Reply:
ah yeah good idea,